evap/results/exporters.py

```diff
@@ -76,13 +76,13 @@ def filter_text_and_heading_questions(self, questions):
 
         return filtered_questions
 
-    def export(self, response, course_types_list, include_not_enough_voters=False, include_unpublished=False):
+    def export(self, response, degrees_list, course_types_list, include_not_enough_voters=False, include_unpublished=False):
```

Comment:
i'd suggest to put the expectation that the two lists are of equal length somewhere in the code. perhaps an assert, or using a list of tuples.

Reply:
k